### DISCUSSION TITLE
How to change OIDC button text

### DISCUSSION THREAD

**Discussion by evanjamesjackson** [pimcore/pimcore#15646] [Q&A / Support] [answered]
Hello - I have just installed the OpenID Connect bundle and followed the documentation here https://pimcore.com/docs/platform/OpenID_Connect/, and I am wondering if it is possible to change the button text. At present it shows 
![image](https://github.com/pimcore/pimcore/assets/5866481/4350a0c5-b9ab-4fc5-b9ef-d4525c3101e6)
but that documentation shows custom button text 
![image](https://github.com/pimcore/pimcore/assets/5866481/c601c719-3b8d-4c99-bd80-b94e4342be46)
however, I can't see any place in the configuration to set this text. 

Line 34 in `src/Resources/views/button-injection/script.js.twig` seems to set it as follows: `link.appendChild(document.createTextNode('{{ ('oidc-button-' ~ service) | trans({}, 'admin') }}'));`

**Comment by dvesh3** [comment, marked as answer]
Hi @evanjamesjackson you can add translation text in Admin UI since the button text is using admin translations 
<img width="590" alt="image" src="https://github.com/pimcore/pimcore/assets/5137917/896dea36-273e-4e1a-a123-e9fb30f588b1">

Also, In the latest version `v1.1.1`, It will automatically translate the button text if no translation is present. https://github.com/pimcore/openid-connect/pull/86

**Reply by evanjamesjackson** [reply]
D'oh - thanks @dvesh3!
